howto/setup-state-store/setup-cassandra.md

```diff
@@ -0,0 +1,98 @@
+# Setup Cassandra 
+
+## Locally
+
+You can run Cassandra locally with th Datastax Docker image:
+
+```
+docker run -e DS_LICENSE=accept --memory 4g --name my-dse -d datastax/dse-server -g -s -k
+```
+
+You can then interact with the server using `localhost:9042`.
```

Comment:
Should we use `127.0.0.1` here instead?

Reply:
This is for docker, best to keep consistent with `localhost` as most samples show.